community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -472,6 +479,115 @@ def update_topology(lkp: util.Lookup) -> None:
         log.debug("Topology configuration updated. Reconfiguring Slurm.")
         util.scontrol_reconfigure(lkp)
 
+
+def delete_reservations(reservation_map: Dict[str, str]) -> None:
```

Comment:
"update by reference" without good reason is considered bad taste.

Reply:
Updated the logic.